public/schemas/searchFilters/all-search.json

```diff
@@ -2,5 +2,22 @@
   "type": "filterCreationSchema",
   "listingType": "all",
   "items": [
+    {
+      "title": {
+        "id": "searchFilters.price",
+        "defaultMessage": "Price"
+      },
+      "type": "filterGroup",
+      "items": [
+        {
+          "type": "price",
+          "searchParameterName": "price.amount",
+          "priceUnit": {
+            "id": "searchFilters.usd",
```

Comment:
Checked with Stan who is a i18n expert and he thinks we don't need to translate currency symbol.